zuul-netflix/src/main/java/com/netflix/zuul/scriptManager/ValidActionEvaluation.java

```diff
@@ -0,0 +1,61 @@
+package com.netflix.zuul.scriptManager;
+
+import java.util.Set;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import static com.google.common.collect.Sets.newHashSet;
+import static java.util.Collections.unmodifiableSet;
+
+final class ValidActionEvaluation {
```

Comment:
@mikeycohen I have deleted my fork I was working in a couple of days ago. I have created a new one and pushed the changes: https://github.com/alexpanov/zuul/commit/db6dda581c81cb77ce2ce7ef2a00159657b5cc57
but only get an option to create a new pull request. How should I go about it?